Bindings/Java/Matlab/Hopper_Device/BuildHopper.m

```diff
@@ -49,22 +49,25 @@
 
 p = inputParser();
 defaultMaxIsometricForce = 4000.0;
-defaultMillardTendonParams = [0.049 28.1 0.67 0.5 0.25];
+defaultOptimalFiberLength = 0.55;
+defaultTendonSlackLength = 0.25;
```

Comment:
I'll push a commit to this branch at some point tonight to update the comment at the top to account for the change in parameters.